tests/features/rhpam/businesscentral-monitoring/rhpam-businesscentral-monitoring.feature

```diff
@@ -1,6 +1,15 @@
 @rhpam-7/rhpam-businesscentral-monitoring-rhel8
 Feature: RHPAM Business Central Monitoring configuration tests
 
+  Scenario: Web console is available
+    When container is ready
+    Then check that page is served
+      | property             | value       |
+      | port                 | 8080        |
+      | path                 | /kie-wb.jsp |
```

Comment:
Not sure but looks like, this is migrated from [test](https://github.com/jboss-container-images/rhdm-7-image/pull/380/files#diff-ff9bf3cb41626d73d5747b9dafb5118fa660bcc17992fd62614b1cfbc50b84b9L9) . The Path is changed?